forms-flow-web/src/components/Form/Item/Edit.js

```diff
@@ -449,377 +249,264 @@ const Edit = React.memo(() => {
+                  <CustomButton
+                    variant="light"
+                    size="md"
+                    label={<Translation>{(t) => t(publisText)}</Translation>}
+                    onClick={handlePublish}
+                    dataTestid={"cancelBtndataTestid"}
+                    ariaLabel={"cancelBtnariaLabel"}
+                  />
                 </div>
               </div>
-            </div>
-
-            <div className="col-lg-6 col-md-6 col-sm-6 col-12 px-3">
-              <div className="d-flex justify-content-between">
-                <div id="form-group-display" className="form-group">
-                  <label htmlFor="form-display" className="control-label fw-bold mb-2">
-                    {t("Display as")}
-                  </label>
-                  <div className="input-group">
-                    <div className="form-check form-check-inline">
-                      <input
-                        data-testid="form-edit-form-display"
-                        className="form-check-input"
-                        type="radio"
-                        name="display"
-                        id="form-radio-form"
-                        value="form"
-                        checked={form.display === "form"}
-                        onChange={(event) => handleChange("display", event)}
-                      />
-                      <label className="form-check-label fw-light" htmlFor="form-radio-form">
-                        {t("Form")}
-                      </label>
+            </Card.Body>
+          </Card>
+          <div className="d-flex mb-3">
+            <div className={`wraper form-wraper ${showLayout ? 'visible' : ''}`}>
+              <Card>
+                <Card.Header>
+                  <div className="d-flex justify-content-between align-items-center" style={{ width: "100%" }}>
+                    <div className="d-flex align-items-center justify-content-between">
+                      <div className="mx-2 builder-header-text">Layout</div>
+                      <div>
+                        <CustomButton
+                          variant="secondary"
+                          size="md"
+                          icon={<HistoryIcon />}
+                          label={<Translation>{(t) => t("History")}</Translation>}
+                          onClick={handleHistory}
+                          dataTestid={"cancelBtndataTestid"}
+                          ariaLabel={"cancelBtnariaLabel"}
+                        />
+                        <CustomButton
+                          variant="secondary"
+                          size="md"
+                          className="mx-2"
+                          icon={<PreviewIcon />}
+                          label={<Translation>{(t) => t("Preview")}</Translation>}
+                          onClick={handlePreview}
+                          dataTestid={"cancelBtndataTestid"}
+                          ariaLabel={"cancelBtnariaLabel"}
```

Comment:
please follow kebab casing while giving datatestid
give a meaningful name for aria label
give translation tag to the arialabeltext (ex : {{(t) => t("Discard Changes")}} )